src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey : string | null;
+};
+
+export class App extends Component<{}, State> {
+  state: State = {
+    pressedKey: null,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.keyPresser);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.keyPresser);
+  }
+
+  keyPresser = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
+  };
```

Comment:
Checklist p.1

Reply:
We don't have a checklist for this task.

Reply:
![image](https://user-images.githubusercontent.com/113321536/232600315-b42c7495-de38-49e9-9dfa-68a4e2f5ab53.png)